lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -110,9 +110,39 @@ def parse_namespace_metadata(namespace_object)
     def get_pod_metadata(namespace_name, pod_name)
       begin
         metadata = @client.get_pod(pod_name, namespace_name)
-        return if !metadata
-        return parse_pod_metadata(metadata)
-      rescue KubeException
+        if !metadata
+          @pod_cache_api_nil_not_found += 1
+        else
+          begin
+            parse_pod_metadata(metadata)
+            @pod_cache_api_updates += 1
+          rescue Exception => e
+            @pod_cache_api_nil_bad_resp_payload += 1
+            nil
+          end
+      rescue KubeException => kube_error
+        @pod_cache_api_nil_error += 1
+        log.debug "Exception encountered fetching pod metadata from Kubernetes API #{@apiVersion} endpoint #{@kubernetes_url}: #{kube_error.message}"
+        nil
+      end
+    end
+
+    def get_namespace_metadata(namespace_name)
```

Comment:
code style... ruby does not typically use 'get' or 'set'

Reply:
it is consistent with the `get_pod_metadata` method.